features/filesystem/storagelite/README.md

```diff
@@ -0,0 +1,92 @@
+## StorageLite
+
+StorageLite is a storage solution, providing a key value store like API of set/get data or a reduced POSIX API of open-write-close and open-read-close. It provides fast write and read functions, power failure resilience, security and built-in backup and factory reset support.
+
+Uses: keeping credentials and cryptographic keys, application settings, application states and small application data to be sent to servers.
+
+Hardware: designed for external NOR flash, it can work also on internal flash and with additional block devices on SD cards.
```

Comment:
FlashSimBlockDevice  is simulation of flash over RAM, SD is already flash memory.
 
> it can work also on internal flash and with additional block devices on SD cards.

Please clarify this. Which additional block device and why

Reply:
Good point @geky. Will add it to documentation. Got a few performance numbers, can send them later today. Maybe we should run your benchmark on this as well.
@deepikabahavani The Flash simulator adaptor applies to any block device not having flash attributes (i.e. erasing). SD is such a device thus requiring the flash simulator adaptor.